### PR TITLE
feat: logger

### PR DESCRIPTION
**Pull request by ycjcl868**
Close https://github.com/umijs/umi-next/issues/25.

image

- [x] logger 实现
- [ ] 测试用例
- [ ] 仓库迁移至 logger

##### Checklist

- [ ] `npm test` passes
- [ ] tests are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

##### Description of change

- any feature?
- close https://github.com/umijs/umi/ISSUE_URL

### CODE REVIEW

File: packages/core/src/Logger/index.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+declare module NodeJS {
```
Comment (sorrycc) [inline packages/core/src/Logger/index.ts]: `Logger/index.ts` 改名为 `Logger/Logger.ts`。

File: packages/core/src/Logger/browser.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import Common from './Common';
```
Comment (sorrycc) [inline packages/core/src/Logger/browser.ts]: 感觉不需要 browser 端，没场景。

**Comment by sorrycc** [comment]
加下用例。

**Comment by ycjcl868** [comment]
ok

File: packages/core/src/Logger/browser.ts (hunk 2 above)
Comment (ycjcl868) [reply packages/core/src/Logger/browser.ts]: 嗯嗯，先去掉

**Pull request merged**